### ISSUE TITLE
Implement instance catalog reader

### ISSUE THREAD

**Issue by fjaviersanchez** [LSSTDESC/gcr-catalogs#66]
For DC2 we want to also check the instance catalogs so a reader should be in place. For now, I'll try to implement it myself.

**Comment by fjaviersanchez** [comment]
I think [this repo](https://github.com/LSSTDESC/sims_GCRCatSimInterface/tree/master/python/desc/sims/GCRCatSimInterface) is related to this issue?

**Comment by yymao** [comment]
@fjaviersanchez Not exactly. That repo has the interface from GCR --> CatSim. And what we need now is the interface from CatSim --> GCR. 

@danielsf and I have discussed the CatSim --> GCR interface and we both agree it should be easy to implement, but obviously neither of us has found time...

**Comment by fjaviersanchez** [comment]
This is started in PR#80
